types/desmos/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
why do you need this? desmos on npm seems to correspond to these types

Reply:
Desmos on npm is at version 1.5.4. These types were produced from the v1.6 documentation, https://www.desmos.com/api/v1.6/docs/index.html.

Reply:
@sandersn the `desmos` package on npm (which is not produced or maintained by Desmos) is versioned `1.5.4`. These types were produced for Desmos 1.6, so they should be versioned as `1.6.0`.

Reply:
Will `@types/desmos` be used in conjunction with the `desmos` package? If so, then the types should cover `1.5.4` or earlier. Definitely Typed is not intended for shipping types ahead of a released version of a package.

If not, then this PR's imports/exports need to change -- right now, they use `import x from 'desmos'`, which only works if `@types/desmos` is matched with `desmos`. If `@types/desmos` is for use from the browser, then everything should be global -- no imports or exports.

Reply:
Got it! It is **not** intended to be used in conjunction with the `desmos` package, I've pushed new changes to put everything in the global namespace.